unstable/lens/string.rkt

```diff
@@ -0,0 +1,43 @@
+#lang racket/base
+
+(provide string-ref-lens
+         string-pluck-lens
+         )
+
+(require fancy-app
+         lens/base/main
+         "join.rkt"
+         )
+(module+ test
+  (require rackunit))
+
+(define (string-ref-lens i)
+  (make-lens
+   (string-ref _ i)
+   (string-set _ i _)))
```

Comment:
That makes sense. Will make issues for implementing various features in terms of isomorphisms once they're added.